src/router.js

```diff
@@ -1,6 +1,17 @@
-//see this link
-//https://github.com/foundersandcoders/modules-challenge/blob/solutions/src/handler.js
+const handler = require('./handlers.js');
 
-//which handler function will be passing
+const router = (req, res) => {
+  const url = req.url;
 
-//routes --> go to handlers.js
+  if (url === '/') {
+    handler.handleHomeRoute(req, res);
+  } else if (url.indexOf('public') !== -1) {
+    handler.handlePublic(req, res);
```

Comment:
This might be better as a whitelist - perhaps we can do this as an enhancement. See #25.